src/mpl/include/mpl_gpu_cuda.h

```diff
@@ -22,4 +23,6 @@ typedef volatile int MPL_gpu_event_t;
 
 #define MPL_GPU_DEV_AFFINITY_ENV "CUDA_VISIBLE_DEVICES"
 
+#define MPL_gpu_device_id_to_handle(...) cuDeviceGet(__VA_ARGS__)
```

Comment:
Rather than an `MPL_gpu_` wrapper, maybe we can directly call `cuDeviceGet`? The usage (in `fi_mr_regattr`) is always inside the `#ifdef MPL_HAVE_CUDA` anyway, which makes it OK to use CUDA-specific calls. If we still want the MPL abstraction, I think that we can name `MPL_cuda_` for CUDA-specific functions.

Reply:
Makes sense. This version just calls `cuDeviceGet` in `ofi_impl.h`.